infoschema/infoschema.go

```diff
@@ -406,6 +408,13 @@ func (is *infoSchema) SetBundle(bundle *placement.Bundle) {
 	is.ruleBundleMap[bundle.ID] = bundle
 }
 
+func (is *infoSchema) AllPlacementPolicies() (policies []*placementpolicy.PolicyInfo) {
+	for _, p := range is.policyMap {
```

Comment:
`policyMutex`?

Reply:
After reading the current code, I found `AllPlacementPolicies` is not needed any more because `PlacementPolicies` did the same work. I'll remove this method and add `PlacementPolicies` to the interface.

But I think it is still safe without `policyMutex` because infoschema is immutable. The happens-before condition can be established for our current implement

Reply:
It is just there in case, IMO. You are right that sometimes we are too cautious to do some great works. But this case is a little bit aggressive for me, especially that other `policy` APIs are following the conract of the mutex.

Reply:
OK. Actually I did not see `policyMutex` and just follow what the function `AllSchemas` did. As other policy APIs use `policyMutex` I think we should follow it either.